mcr.cpp

```diff
@@ -26,19 +30,24 @@ int main(){
                 cout << "Player 1: ";
             else
                 cout << "Player 2: ";
-            cout << "Which cell to mark? i:[1..3], j:[1..3]: "; 
+            cout << "Which cell to mark? i:[0..2], j:[0..2]: "; 
             cin >> i >> j;
             if (turn == false)
-               game[i][j] = 'X';
+               game[i][j] = player1;
             else 
-               game[i][j] = 'O';
+               game[i][j] = player2;
             if (isWin(game)){
                 cout << "Win!" << endl;
                 break; // need to terminate the problem
             }
         }
-    if (i==3) // all celles with i=0 to 2 have been inputted above but no winner yet
-       cout << "Tie!" << endl;
+    bool isNotFull = false;
+    for (i=0; i<3; i++){
+        for (j=0; j<3; j++)
+            if (game[i][j] == ' ') isNotFull = true;
+    }
+    if (isNotFull)
```

Comment:
the condition is incorrect

Reply:
updated, please reference to commit [fix: update the incorrect condition for tie](https://github.com/crapper/modern-code-review/pull/1/commits/4fecaa19ba815b2907023cff04a45f7cdb31e806)